Research/Research/RSDFontRules.swift

```diff
@@ -102,6 +102,19 @@ open class RSDFontRules  {
             return RSDFont.systemFont(ofSize: 80, weight: .light)
         }
     }
+    
+    /// Returns the font to use for a given button type.
+    ///
+    /// - parameter buttonType: The button type for the font.
+    /// - returns: The font to use for this button text.
+    open func font(for buttonType: RSDDesignSystem.ButtonType) -> RSDFont {
+        switch buttonType {
+        case .primary:
+            return RSDFont.systemFont(ofSize: 20, weight: .bold)
+        case .secondary:
+            return RSDFont.systemFont(ofSize: 18, weight: .bold)
+        }
+    }
```

Comment:
I will confirm with Stockard tomorrow that this is the desired behavior.

Reply:
See https://www.figma.com/file/nvoSigSxbFuWzGXgUZAf8M/DigitalHealth_DesignSystem-Master?node-id=3125%3A322

Reply:
So... Stockard and I discussed how fonts for the rounded buttons should *not* be dynamic b/c that messes with the sizing of the button, and that the text on the button can be made accessible by turning on voice-over instead.

Toward that end, we talked about introducing a method that is more generic. Namely, doing this:

```
    /// Returns a font in the given size and weight in the font family specified for this design. Typically, you will
    /// want to use `font(for textType: RSDDesignSystem.TextType)` instead for copy and dynamic text. This method should
    /// only be used where the design calls for a specific size to match the graphic design of the view.
    ///
    /// - parameters:
    ///     - fontSize: The font size.
    ///     - weight: The font weight.
    /// - returns: The font to use for this size and weight.
    open func font(ofSize fontSize: CGFloat, weight: RSDFont.Weight) -> RSDFont {
        return RSDFont.systemFont(ofSize: fontSize, weight: weight)
    }
```

And then use *this* method for elements that should be a statically defined size/weight (such as the rounded button).

The frameworks/applications can then set the font family or whatever to override the base class (although eventually we want to include Lato in this bundle.) 

Also, this leaves room in the designs to eventually accommodate vision impaired folks who prefer larger text.